bin/genome-env-next

```diff
@@ -0,0 +1,432 @@
+#!/bin/bash
+
+# if this was found via PATH as opposed to a path to executable
+THIS="$(basename "$0")"
+if test "$THIS" == "$0"
```

Comment:
If the script is found via $PATH, won't $0 be an absolute path? I'm reading this as a test to see if we were called with a path or not. I'm not sure if that's what will actually happen though (please correct me if I am wrong!). If you ran the command from the dir it was in as "bash genome-env-next" then you would get $0 == "genome-env-next". I'm not sure how else you'd get that though.